test/fixtures.js

```diff
@@ -5,21 +5,21 @@ import Task, {
   drainTasksForTesting
 } from '../src/tasks';
 
-import {createStore, applyMiddleware} from 'redux';
+import {createStore, applyMiddleware, type Reducer, type Store} from 'redux';
 
-export const ECHO_TASK = taskCreator(
+export const ECHO_TASK = taskCreator<string, _, _>(
```

Comment:
You can use a literal underscore to declare a generic?

Reply:
> If you only want to specify some of the type arguments, you can use _ to let flow infer a type for you ([docs](https://flow.org/en/docs/types/generics/))

TIL. Strange they don't use `*` for this

Reply:
I think `*` is something else but  🤷‍♂️